scripts/local/run_stop.sh

```diff
@@ -5,11 +5,6 @@
 set -e # Stop on first error
 source ./scripts/utils.sh
 
-# Set ARCH env so as a container executes without issues in a portable way
-# Should be amd64 for linux/macos x86 hosts, and arm64 for macos M1
-# It is referenced in the docker composer yaml, and then passed as a Dockerfile ARG
-setARCH
```

Comment:
why do we need to remove this?

Reply:
ARCH isn't used in this file. I think we should really only be using `setARCH` to set the env var needed for the dockerfiles to run correctly. The other functions in `utils.sh` should probably check `$(uname -m)` each time so we're not relying on having `setARCH` be called from every script.